### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dariusshop**
Updated README.md file provide more context about the package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 midnight/package-cohesion
 =========================
-This is a script that checks the cohesion of your composer package.
+This script checks how cohesive your Composer package is. It shows you file dependency clusters. If you've got a single cluster, your all good. Your package is very cohesive. If you've got one big cluster and several small ones, you should be fine. If you've got two or more larger clusters, you probably want to split your package into several smaller packages.
```
Comment (muli) [inline README.md:3]: May I suggest some grammar and punctuation changes?

```suggestion
This script checks how cohesive your Composer package is, i.e., your file dependency clusters.

If you've got a single cluster, you're all good, your package is very cohesive.
If you've got one big cluster and several small ones, you should be fine.
If you've got 2 or more larger clusters, you probably want to split your package into several smaller packages.
```

**Comment by MidnightDesign** [comment]
Perfect, thank you very much!

File: README.md (hunk 1 above)
Comment (MidnightDesign) [reply README.md:3]: Great changes! Could you open a separate PR please?

Comment (muli) [reply README.md:3]: @MidnightDesign, there you go: https://github.com/MidnightDesign/package-cohesion/pull/2.

**Pull request merged**